cms/static/js/views/utils/create_library_utils.js

```diff
@@ -5,14 +5,19 @@ define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
     function ($, _, gettext, ViewUtils) {
         "use strict";
         return function (selectors, classes) {
-            var validateTotalKeyLength, setNewLibraryFieldInErr, hasInvalidRequiredFields,
-                createLibrary, validateFilledFields, configureHandlers;
+            var toggleSaveButton, validateTotalKeyLength, setNewLibraryFieldInErr,
+                hasInvalidRequiredFields, createLibrary, validateFilledFields, configureHandlers;
 
             var validateRequiredField = ViewUtils.validateRequiredField;
             var validateURLItemEncoding = ViewUtils.validateURLItemEncoding;
 
             var keyLengthViolationMessage = gettext("The combined length of the organization and library code fields cannot be more than <%=limit%> characters.");
 
+            toggleSaveButton = function (is_enabled) {
```

Comment:
@cahrens Yes, there is other code that is duplicated. I'll take a look at refactoring that.